libs/s25main/Loader.cpp

```diff
@@ -280,7 +288,7 @@ bool Loader::LoadSounds()
 
 bool Loader::LoadFonts()
 {
-    if(!LoadFile(config_.ExpandPath("<RTTR_RTTR>/LSTS/fonts.LST"), GetPaletteN("pal5")))
+    if(!Load(config_.ExpandPath("<RTTR_RTTR>/LSTS/fonts.LST"), GetPaletteN("pal5")))
```

Comment:
should the path here possibly also added to the FILE_PATHS or at least as a global constant in the same file as the rest of the paths, that way we have them in one place.

Reply:
I am very much against that list of files because it's incredibly hard to work with especially as the indices into some array are used. This leads to things like `Load(FILEPATHS[2]); // foo.lst` which is bad for 2 reasons: Duplication of the name (in array and comment) and it can possibly become out of sync.

In the resourceLoading branch I have I started by adding the folders with resources to a registry and then load resources by plain resourceIDs (filename w/o any extension) which avoids the issues.

So I'd wait till we introduce something like that and not move even more into that array than there already is.

Reply:
yes but perhaps we can add some kind of FILE_PATH_FONTS_LST as a constant

Reply:
I don't see the advantage. IIRC this is only used once and the name of the constant is almost as long as the path. So I'd keep the changeset low and fix all those paths later.

Reply:
the advantage is that we have all paths together in one file - so if we need to change them, we have them at one place

Reply:
Question then: Shouldn't that be a separate PR? This PR doesn't introduce new path constants so IMO this whole discussion is valuable but out of scope of this PR.

Reply:
I like that commit. its okay if its a separate PR though

Reply:
Ok. Removed it from this PR so this can be merged now